test/e2e/utils_test.go

```diff
@@ -90,6 +92,97 @@ func checkIfAllMemberClustersHaveJoined() {
 	}
 }
 
+// createResourcesForFleetGuardRail create resources required for guard rail E2Es.
+func createResourcesForFleetGuardRail() {
```

Comment:
Hi Arvind! Sorry if I missed anything, but these two resources do not seem to be used in any place.

Reply:
They are being used here https://github.com/Azure/fleet/pull/566/files#diff-cbecd05d3d19bc074c3463404c4c2d4af4f6b2dde9a15e0e587df4bd97e3a13fR176